py-polars/polars/series/datetime.py

```diff
@@ -1087,65 +1089,80 @@ def replace_time_zone(self, time_zone: str | None) -> Series:
         ----------
         time_zone
             Time zone for the `Datetime` Series. Pass `None` to unset time zone.
+        is_earliest
+            If localizing an ambiguous datetime (say, due to daylight saving time),
+            determine whether to localize to the earliest datetime or not.
+            If None (the default), then ambiguous datetimes will raise.
 
         Examples
         --------
         >>> from datetime import datetime
-        >>> start = datetime(2020, 3, 1)
-        >>> stop = datetime(2020, 5, 1)
-        >>> date = pl.date_range(start, stop, "1mo", time_zone="UTC")
-        >>> date
-        shape: (3,)
-        Series: '' [datetime[μs, UTC]]
-        [
-                2020-03-01 00:00:00 UTC
-                2020-04-01 00:00:00 UTC
-                2020-05-01 00:00:00 UTC
-        ]
-        >>> date.dt.epoch("s")
-        shape: (3,)
-        Series: '' [i64]
-        [
-                1583020800
-                1585699200
-                1588291200
-        ]
-        >>> date = date.dt.convert_time_zone(time_zone="Europe/London").alias("London")
-        >>> date
-        shape: (3,)
-        Series: 'London' [datetime[μs, Europe/London]]
-        [
-            2020-03-01 00:00:00 GMT
-            2020-04-01 01:00:00 BST
-            2020-05-01 01:00:00 BST
-        ]
-        >>> # Timestamps have not changed after convert_time_zone
-        >>> date.dt.epoch(time_unit="s")
-        shape: (3,)
-        Series: 'London' [i64]
-        [
-                1583020800
-                1585699200
-                1588291200
-        ]
-        >>> date = date.dt.replace_time_zone(time_zone="America/New_York").alias("NYC")
-        >>> date
-        shape: (3,)
-        Series: 'NYC' [datetime[μs, America/New_York]]
-        [
-            2020-03-01 00:00:00 EST
-            2020-04-01 01:00:00 EDT
-            2020-05-01 01:00:00 EDT
-        ]
-        >>> # Timestamps have changed after replace_time_zone
-        >>> date.dt.epoch(time_unit="s")
-        shape: (3,)
-        Series: 'NYC' [i64]
-        [
-            1583038800
-            1585717200
-            1588309200
-        ]
```

Comment:
this docstring has diverged quite a lot from the respective one in py-polars/polars/expr/datetime.py ... I'm suggesting to just use the one from py-polars/polars/expr/datetime.py , but with an extra example of how to use `is_earliest` (I don't think users tend to think about the underlying timestamps anyway?)